Makefile

```diff
@@ -3,9 +3,11 @@ INSTALL_ARGS :=	# todo: add --break-system-packages if ubuntu
 # Get git version information for make install
 GIT_DESCRIBE := $(shell git describe --always --long --dirty)
 
+TARGET := $(shell rustc -vV | grep 'host:' | cut -d' ' -f2 | sed 's/apple-darwin/unknown-linux-musl/')
+
 .PHONY: init
 init:
-	cd virtme_ng_init && cargo install --path . --root ../virtme/guest
+	cd virtme_ng_init && cargo install --path . --root ../virtme/guest --target $(TARGET) --config "target.$(TARGET).linker = \"rust-lld\""
```

Comment:
This seems to break the build on Linux, since rust-lld isn't available. Maybe set the linker only if the binary exists in the PATH?

Reply:
Hm... I don't see it in my system (tested both on Ubuntu 24.04 and CachyOS). Checking if it exists in the path shouldn't make the Makefile logic too complex, something like:
```
ifeq ($(shell command -v rustc-lld 2> /dev/null),)
    INSTALL_CMD = cd virtme_ng_init && cargo install --path . --root ../virtme/guest --target $(TARGET)
else
    INSTALL_CMD = cd virtme_ng_init && cargo install --path . --root ../virtme/guest --target $(TARGET) --config "target.$(TARGET).linker = \"rust-lld\""
endif
```